.ci/install_runtime.sh

```diff
@@ -33,7 +34,9 @@ NEW_RUNTIME_CONFIG="${PKGDEFAULTSDIR}/configuration.toml"
 clone_build_and_install "github.com/kata-containers/runtime"
 
 # Check system supports running Kata Containers
-kata-runtime kata-check
+if [ "$ID" == "ubuntu" ]; then
+	kata-runtime kata-check
```

Comment:
why only check kata support for ubuntu ? could you add a comment if this is only need for this.